commons-dlp/src/main/java/org/exoplatform/commons/dlp/domain/DlpPositiveItemEntity.java

```diff
@@ -0,0 +1,66 @@
+package org.exoplatform.commons.dlp.domain;
+
+import org.exoplatform.commons.api.persistence.ExoEntity;
+
+import javax.persistence.*;
+import java.util.Calendar;
+
+/**
+ * Entity for Dlp positive item. Created by The eXo Platform SAS Author :
+ * eXoPlatform exo@exoplatform.com
+ */
+@Entity(name = "DlpPositiveItemEntity")
+@ExoEntity
+@Table(name = "DLP_POSITIVE_ITEMS")
+@NamedQueries({
+        @NamedQuery(name = "DlpPositiveItemEntity.getDlpItemByUUID",
+                query = "SELECT q FROM DlpPositiveItemEntity q WHERE q.uuid = :uuid")
+})
+
+public class DlpPositiveItemEntity {
+
+    @Id
+    @SequenceGenerator(name = "SEQ_ITEM_ID", sequenceName = "SEQ_ITEM_ID")
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "SEQ_ITEM_ID")
+    @Column(name = "ITEM_ID")
+    private String id;
+
+    @Column(name = "ITEM_UUID")
```

Comment:
```suggestion
    @Column(name = "ITEM_REFERENCE")
```
I prefer ITEM_REFERENCE, because for activities for exemple it will not be a uuid